<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>221</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>安装APK</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>路径不要包含中文</string>
    </property>
    <property name="placeholderText">
     <string>路径不要包含中文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pickFileBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
